### DISCUSSION TITLE
[Feature] Keyboard shorcut PageUp/PageDown

### DISCUSSION THREAD

**Discussion by chaminaud** [immich-app/immich#6264] [Feature Request]
### The feature

Using Immich I wanted to use Page UP and Page Down keys to scroll, and its seemed not implemented yet.

In fact, I need to click in the thumbnails area for the Keys to be taken into account by the browser itself, because the focus is in the good section after that.

It will be nice to bind key Events to scroll using keyboard without any previous focus setting.

thank you, for the great app! 

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by lyz-code** [comment]
Possible duplicate of https://github.com/immich-app/immich/discussions/6202
